locale/en_US/manager.xml

```diff
@@ -280,7 +280,7 @@
 	<message key="manager.setup.homepageImageInvalid">Invalid homepage image format or upload failed. Accepted formats are .gif, .jpg, or .png.</message>
 	<message key="manager.setup.homeTitleImageInvalid">Invalid homepage header title image format or upload failed. Accepted formats are .gif, .jpg, or .png.</message>
 	<message key="manager.setup.layout.theme">Theme</message>
-	<message key="manager.setup.layout.themeDescription">Choose a theme from the available set of theme plugins.</message>
+	<message key="manager.setup.layout.themeDescription">Select a theme for your website. Only themes installed from the plugins settings are listed here.</message>
```

Comment:
This new language is confusing to me. Maybe "New themes may be added from the Plugins setup area"?